### PR TITLE
Experimental settings: Linux/macOS - Hardware + Volumes tab

### PR DESCRIPTION
**Pull request by s0nea**
Screenshot_2023-04-24_19-12-11
Screenshot_2023-05-03_16-02-22

Fixes https://github.com/rancher-sandbox/rancher-desktop/issues/4333

### CODE REVIEW

**Review by rak-phillip — COMMENTED** [review]
I left some comments on spacing

File: pkg/rancher-desktop/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -302,6 +302,47 @@ virtualMachine:
...
+              legend: Security Model
+              tooltip: Security model used for the export path
+              options:
+                passthrough: passthrough
+                'mapped-xattr': mapped-xattr
+                'mapped-file': mapped-file
+                none: none
+        virtiofs:
+          label: virtiofs
+          description: Exposes the filesystem by using apple Virtualization.Framework shared directory (uses virtio-fs)
```
Comment (rak-phillip) [inline pkg/rancher-desktop/assets/translations/en-us.yaml:344]: Apple should be a proper noun

```suggestion
          description: Exposes the filesystem by using Apple Virtualization.Framework shared directory (uses virtio-fs)
```

Comment (rak-phillip) [inline pkg/rancher-desktop/assets/translations/en-us.yaml:344]: I just want to confirm that `Virtualization.Framework` correct? I'm unfamiliar and want to double check.

**Review by rak-phillip — CHANGES_REQUESTED** [review]
Looks great! I only have one minor bit of feedback and a question.

File: pkg/rancher-desktop/assets/translations/en-us.yaml (hunk 1 above)
Comment (s0nea) [reply pkg/rancher-desktop/assets/translations/en-us.yaml]: Oh, yes, of course. Thanks for pointing it out!

**Pull request merged**